service/etc/agama.yaml

```diff
@@ -20,6 +20,88 @@ web:
   ssl_cert: null
   ssl_key: null
 
+ALP-Dolomite:
```

Comment:
why it needs to be moved? ( sorry for question if it is in commit message, but I failed to find way to see commit belonging to final change in pull request and there are too many commits )

Reply:
I simply reorganized the file a bit to define the products in the same order they are listed above. Sorry for the confusion.